sdk/personalizer/ai-personalizer-rest/test/public/configurationTests.spec.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder, delay, env } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import createPersonalizerClient, {
+  PersonalizerClient,
+  PolicyContract,
+  PolicyContractOutput,
+  PolicyUpdateParameters,
+  ServiceConfiguration,
+  ServiceConfigurationOutput,
+  ServiceConfigurationUpdateParameters,
+  isUnexpected,
+} from "../../src";
+
+describe("Configuration Tests", () => {
+  let recorder: Recorder;
+  let client: PersonalizerClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+    client = createPersonalizerClient(
+      env["PERSONALIZER_ENDPOINT_SINGLE_SLOT"] ?? "",
+      {
+        key: env["PERSONALIZER_API_KEY_SINGLE_SLOT"] ?? "",
+      },
+      recorder.configureClientOptions({})
+    );
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("update configuration test", async function () {
+    const configuration: ServiceConfigurationUpdateParameters = {
+      body: {
+        rewardAggregation: "average",
+        modelExportFrequency: "PT3M",
+        defaultReward: 1.0,
+        rewardWaitTime: "PT4H",
+        explorationPercentage: 0.3,
+        logRetentionDays: 100,
+        learningMode: "Online",
+      },
+    };
+
+    const updatedConfiguration = await updateConfigurationAsync(client, configuration);
+    assertServiceConfigurationEquals(updatedConfiguration, configuration.body);
+    await delay(30 * 1000);
+    const newConfiguration = await getConfigurationAsync(client);
+    assertServiceConfigurationEquals(newConfiguration, configuration.body);
+  });
+
+  it("update policy test", async function () {
+    const policy: PolicyUpdateParameters = {
+      body: {
+        name: "app1",
+        arguments:
+          "--cb_explore_adf --quadratic GT --quadratic MR --quadratic GR --quadratic ME --quadratic OT --quadratic OE --quadratic OR --quadratic MS --quadratic GX --ignore A --cb_type ips --epsilon 0.2",
+      },
+    };
+    const updatedPolicy = await updatePolicyAsync(client, policy);
+    assertPolicyEquals(updatedPolicy, policy.body);
+    await delay(30 * 1000);
+    const newPolicy = await getPolicyAsync(client);
+    assertPolicyEquals(newPolicy, policy.body);
+  });
+
+  it("reset policy test", async function () {
+    const expectedPolicy: PolicyContractOutput = {
+      name: "app1",
+      arguments: "--cb_explore_adf --epsilon 0.2 --power_t 0 -l 0.001 --cb_type mtr -q ::",
+    };
+    const resetPolicy = await resetPolicyAsync(client);
+    assertPolicyEquals(resetPolicy, expectedPolicy);
+    await delay(30 * 1000);
+    const newPolicy = await getPolicyAsync(client);
+    assertPolicyEquals(newPolicy, expectedPolicy);
+  });
+});
+
+async function getConfigurationAsync(
```

Comment:
Can you please remove the Async suffix for these functions? While it is a common convention in C# it is not idiomatic to JS